tests/rbd_mirror/test_rbd_mirror_journal_to_snap.py

```diff
@@ -5,6 +5,41 @@
 log = Log(__name__)
 
 
+def test_write_from_secondary(self, mirrormode, imagespec):
+    """Method to verify write operation should not be performed from secondary site,
+    as mirrored image got locked from primary site.
+
+    Args:
+        self: mirror2 object
+        mirrormode: type of mirroring journal or snapshot
+        imagespec: poolname + "/" + imagename
+    Returns:
+        0 - if test case pass
+        1 - it test case fails
+    """
+    out, err = self.exec_cmd(
```

Comment:
Please use the benchwrite method present in rbd_mirror_utils for this instead of executing your own command again.
Let's reuse existing common methods.

Reply:
If you need that method to return both out and err modify that method to return the same as an optional return whenever output is requested.

You can take kw["output"] = True as input to benchwrite method and if it is set to true then you return both out and err, something similar to exec_cmd method in rbd_utils.

Reply:
Initially me to thought the same way and tried but https://github.com/red-hat-storage/cephci/blob/master/tests/rbd_mirror/rbd_mirror_utils.py#L62
as bench write is executing through `long running` it just return `out` no `error`
Hence used the direct rbd bench.